File: core/src/nl/t64/game/rpg/components/conversation/PhraseIdContainer.java
package nl.t64.game.rpg.components.conversation;

import java.util.HashMap;
import java.util.Map;


public class PhraseIdContainer {

    private final Map<String, String> currentPhraseIds;

    PhraseIdContainer() {
        this.currentPhraseIds = new HashMap<>();
    }

    String getPhraseId(String conversationId) {
        return currentPhraseIds.get(conversationId);
    }

    void setPhraseId(String conversationId, String phraseId) {
        currentPhraseIds.put(conversationId, phraseId);
    }

    int getSize() {
        return currentPhraseIds.size();
    }

}

File: core/src/nl/t64/game/rpg/components/conversation/PhraseIdContainer.kt
package nl.t64.game.rpg.components.conversation


class PhraseIdContainer {

    private val currentPhraseIds: MutableMap<String, String> = HashMap()
    val size: Int get() = currentPhraseIds.size

    fun getPhraseId(conversationId: String): String {
        return currentPhraseIds[conversationId]!!
    }

    fun setPhraseId(conversationId: String, phraseId: String) {
        currentPhraseIds[conversationId] = phraseId
    }

}
